runtime/src/accounts.rs

```diff
@@ -251,9 +252,6 @@ impl Accounts {
                     }
 
                     if solana_sdk::sysvar::instructions::check_id(key) {
-                        if message.is_writable(i, demote_program_write_locks) {
```

Comment:
Why was this removed?

Reply:
@jstarry pointed out that, because sysvar write locks are being demoted, this will never return true `if solana_sdk::sysvar::instructions::check_id(key)`.

Reply:
So just tidying up.